clutchdevteam/docs-place
# Carousel

Creates a BaseCarousel component (reusable and re-stylable) and a Carousel component as the storyblok "component.

### How to test
- Pull this branch down and check out the admissions page, the carousel is in the middle
